servers/src/main/java/tachyon/master/block/meta/MasterWorkerInfo.java

```diff
@@ -68,8 +72,9 @@ public MasterWorkerInfo(long id, NetAddress address) {
     mStartTimeMs = System.currentTimeMillis();
     mLastUpdatedTimeMs = System.currentTimeMillis();
     mIsRegistered = false;
-    mTotalBytesOnTiers = Collections.emptyList();
-    mUsedBytesOnTiers = Collections.emptyList();
+    mStorageTierAssoc = new StorageTierAssoc(new ArrayList<String>());
+    mTotalBytesOnTiers = new HashMap<String, Long>();
+    mUsedBytesOnTiers = new HashMap<String, Long>();
```

Comment:
Not mStorageTierAssoc, but the others can be called conceivably if we added the worker, and, before we register it, we query the total/used bytes from the web interface or something.